### ISSUE TITLE
GodsEye Errors

### ISSUE THREAD

**Issue by Marc-BTC** [Nort721/GodsEye#295]
A bug report/issue report should contain:

1) GodsEye vBuild 40 RELEASE
2) Paper
3) ProtocolLib v4.6.0-SNAPSHOT-b483
4) Server log
5) & 6):

My console gets sometimes spammed with this error. 

[GodsEye]: Plugin GodsEye vBuild 40 RELEASE generated an exception while executing task 2953

Godseye caused the server to drop to 9 TPS. after I deativated it
the TPS are normal again at 20, as always.

Here are some other errors wich may are responsible too for the TPS drop:
https://ibb.co/h8JdCv4

I had to disable the movement section and invent actions completely (long time ago before the TPS thing) (But on vBuild 40 RELEASE)

I got massive spam of player "Movement" and it was all of the time false positive.
Example: Some of them were sneaking on wood slaps while building their house, and this error gets spammed.

Basically most of the time (evening) there are 7-12 players on the server and if I get any alerts for suspicious activity its 99% false positive. I´ll get like houndrets of false positives and then maybe one not false positive. (But me and the mods can´t check this everytime cause of the incredible amount of false positives)

The false positives occur basically on all checking methods, like movement, speed, killaura, range and so on.

I´ve "HackedServer - FORGE MODS & CLIENTS detector" also installed on the server to disable that people can even join with forge or fabric api. Besides that the false positives occour on players who dont cheat to 100%. Even on myself if I deop me.

Here I pasted some of the logs of /Godseye/logs
https://www.protectedtext.com/2039z48f?4XXEfecmYrs3XV

Thx.

**Comment by Nort721** [comment]
The error you have posted should be fixed for the next version, from looking at your logs seems like you don't have much problems with movement, your problems were with other checks that have been fixed in the last hotfix update, make sure to stay on the latest version, it's most likely that the TPS drops were from the error spam which should be solved now

So update to the latest version and once the next update is released the error spam should be fixed with it, even though I never had this issue before so yeah

**Comment by Marc-BTC** [comment]
Thx Sir. Very fast response :)
